.markdownlint.yml

```diff
@@ -1,13 +1,16 @@
 ---
-# https://github.com/DavidAnson/markdownlint
-# https://github.com/igorshubovych/markdownlint-cli
-# https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
+# https://github.com/DavidAnson/markdownlint-cli2
+# https://github.com/DavidAnson/markdownlint/blob/main/README.md#rules--aliases
+# https://github.com/DavidAnson/markdownlint/blob/main/schema/.markdownlint.yaml
 
 default: true
 
 MD002: false # deprecated
-MD003: {style: atx}
-MD004: {style: "asterisk"}
+MD003:
+  style: "atx"
+MD004:
+  style: "asterisk"
 MD006: false # deprecated
 MD013: false # we use https://sembr.org that could lead to long lines
-MD029: {style: ordered}
+MD029:
+  style: "ordered"
```

Comment:
Please keep formatting the same

Reply:
this is the same formatting just the syntax is changed according to the newer version

Reply:
Both old and new lines represent the same YAML object; the former just uses JSON-like inline formatting. Let's keep it that way to minimize unrelated changes in this PR.